pkgs/development/ocaml-modules/hack_parallel/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib
+, buildDunePackage
+, fetchFromGitHub
+, ocaml
+, core
+, ppx_deriving
+, ppxlib
+, sexplib
+}:
+
+buildDunePackage rec {
+  pname = "hack_parallel";
+  version = "0.1.1";
+
+  src = fetchFromGitHub {
+    owner = "rvantonder";
+    repo = "hack-parallel";
+    rev = "v${version}";
+    sha256 = "051hbnj4lvvfdb3vs14apl7kwpdn9rbalnllfdc4xnhv3rzzxsyk";
+  };
+
+  buildInputs = [ core ppx_deriving ppxlib sexplib ];
+
+  patches = [ ./dune-static-libs.patch ];
```

Comment:
Nitpick: technically it's a diff, not a patch :) Compare:

- https://github.com/NixOS/nixpkgs/commit/228627d50e1be526aa87a97c282ecf9dcbacc6eb.diff
- https://github.com/NixOS/nixpkgs/commit/228627d50e1be526aa87a97c282ecf9dcbacc6eb.patch.